src/main/javascript/view/OperationView.js

```diff
@@ -313,7 +313,7 @@ SwaggerUi.Views.OperationView = Backbone.View.extend({
       if (isFileUpload) {
         return this.handleFileUpload(map, form);
       } else {
-        return this.model['do'](map, opts, this.showCompleteStatus, this.showErrorStatus, this);
+        return this.model.execute(map, this.options.swaggerOptions, this.showCompleteStatus, this.showErrorStatus);
```

Comment:
From what I can tell, `this.options` is undefined and should be `this.opts`  _([src/main/javascript/view/OperationView.js#L16](https://github.com/3scale/swagger-ui/blob/op-execute/src/main/javascript/view/OperationView.js#L16))_.  Also, should you change `this.options.swaggerOptions` to `this.options.swaggerOptions || {}` just to be safe?